code/modules/tier_weapon_spawns/tier_spawn.dm

```diff
@@ -0,0 +1,71 @@
+/obj/effect/landmark/weapon_spawn/proc/spawn_weapon(/obj/item/weapon/weapon_to_spawn)
+	new weapon_to_spawn (src)
+
+/obj/effect/landmark/weapon_spawn/proc/spawn_associated_ammo(obj/item/weapon/gun/gun_to_spawn)
+	for(var/i in 1 to 3) //hardcoded 3 mags.
+		new gun_to_spawn.current_mag
+
+//pistols and knives
+/obj/effect/landmark/weapon_spawn/proc/tier_1()
+	weapon_list = list(	/obj/item/weapon/gun/pistol/standard_pistol,
+						/obj/item/weapon/gun/pistol/m4a3,
+						/obj/item/weapon/combat_knife,
+						/obj/item/weapon/throwing_knife
+						)
```

Comment:
Move the list away from the proc and into the object definition.

Reply:
done